### DISCUSSION TITLE
Welcome to community matters Discussions!

### DISCUSSION THREAD

**Discussion by HaireLab** [HaireLab/community-stress-gradients#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by megkrawchuk** [comment]
Hi everyone.
Dropping in here to provide an overview of what I'm thinking for the next stages of the community/traits datasets. Based on our last discussion about available traits, and highlighting from Claire and Emma that it would be a very big lift to fill in the entire traits matrix. And the way that I was thinking of characterizing each plot would require that all species be "attributed", then clustering based on those attributes (e.g., k-means or PAM based on function/life forms cover, rather than species cover).
So here's what I'm thinking:
1. Each species at each plot is assigned to one of nine basic functional groups, based on our existing growth form/habit data, duration, plus the new phenology/leaf type traits data. These are the most widely available traits, with relatively easy coverage for all species in our dataset:
- annual forb
- annual graminoid
- perennial graminoid
- perennial forb
- shrub deciduous
- shrub evergreen
- subshrub
- tree deciduous
- tree evergreen

2. Cover is summarized for each plot using the nine functional groups. So that each plot has an estimate of composition of each general group. These groups are broadly faithful to hypotheses we generated in the shared document. But we can also try others (simplify, expand) as needed.
3. We can then use k-means or PAM to cluster our plots together, like with like, according to similarities in their functional group composition/cover. I'm imagining six-nine groups from the clustering?
4. Now that we have these broader groups, we can look at the dominant species for each plot, according to these functional groups, and further attribute them with e.g., n-fixers, rooting architecture/depth, mycorrhizal assoc., exotic, focusing ONLY on the dominant species. This will make it a much easier lift. So this is a two part functional group assessment. These additional traits can be analyzed descriptively for particular groupings/sites, and used to interpret the outcomes.
4. The groups would used to label each plot, and that label would be used in the statistical analysis as the "community" categorical term.
5. We will only focus on plots burned at high severity. The refugia plots will be omitted.

I will post some examples here once I've played a bit with the data.
Feedback welcome!

**Reply by megkrawchuk** [reply]
Go check out the first round of results over in the "Community types" discussion thread:
https://github.com/HaireLab/community-stress-gradients/discussions/6#discussion-3439148
